### PR TITLE
ARM64 support for ECS and Fargate

### PR DESCRIPTION
**Pull request by dmitry-livchak**
Support explicit runtime platform architecture selection for ECS and Fargate clusters

### CODE REVIEW

File: dask_cloudprovider/aws/ecs.py [hunk 1]
```diff
@@ -754,6 +759,7 @@ def __init__(
         mount_volumes_on_scheduler=False,
         **kwargs,
     ):
+        self._cpu_architecture = cpu_architecture
```
Comment (jacobtomlinson) [inline dask_cloudprovider/aws/ecs.py:762]: Could we add a case check here?

It looks like valid values are `X86_64` and `ARM64`. However the lowercase `x86_64` and `arm64` are also very commonly used in other situations (this is what `uname -m` will show).

Perhaps we want to either explicitly set this with `.upper()` or check the value against known good values (although if more are added in the future this adds a maintenance burden).

**Review by jacobtomlinson — COMMENTED** [review]
Thanks for raising this!

File: dask_cloudprovider/aws/ecs.py (hunk 1 above)
Comment (dmitry-livchak) [reply dask_cloudprovider/aws/ecs.py]: .upper() it is!

Comment (dmitry-livchak) [reply dask_cloudprovider/aws/ecs.py]: @jacobtomlinson I also moved it up in line with order of the constructor fields. Happy to proceed?

**Review by jacobtomlinson — APPROVED** [review]
This is great, thank you! The RTD failure is unrelated so happy to merge.

**Pull request merged**